ansible/roles/mn-evo-services/templates/tendermint/genesis.json.j2

```diff
@@ -14,11 +17,11 @@
     },
     "validator": {
       "pub_key_types": [
-        "ed25519"
+        "bls12381"
       ]
-    }
+    },
+    "version": {}
   },
-  "initial_core_chain_locked_height": {{ platform_initial_core_chain_locked_height | int }},
   "validators": [
     {% for masternode in groups.masternodes[:tenderdash_max_validators_count] %}
```

Comment:
Don't we decided to add just a dummy one?

Reply:
Should I add the dummy value in `group_vars/all` or in `dash-network-configs`? Or just delete the template in `genesis.json.j2` and put in an integer? And what should the value be? I had trouble yesterday with blocks coming too fast, and I thought it might be creating fast blocks until it reaches this height?